developers/transforms/distributions/index.qmd

```diff
@@ -0,0 +1,314 @@
+For example, in the normal distribution, half of the probability density is between $-\infty$ and $0$, and half is between $0$ and $\infty$.
+When exponentiated (i.e. in the log-normal distribution), the first half of the density is mapped to the interval $(0, 1)$, and the second half to $(1, \infty)$.
+
+This 'explanation' on its own does not really mean much, though.
+A perhaps more useful approach is to not talk about _probability densities_, but instead to make it more concrete by relating them to actual _probabilities_.
+If we think about the normal distribution as a continuous curve, what the probability density function $p(x)$ really tells us is that: for any two points $a$ and $b$ (where $a \leq b$), the probability of drawing a sample between $a$ and $b$ is the corresponding area under the curve, i.e.
+
+$$\int_a^b p(x) \, \mathrm{d}x.$$
+
+For example, if $(a, b) = (-\infty, \infty)$, then the probability of drawing a sample from the entire distribution is 1.
+
+Let's say that the probability density function of the log-normal distribution is $q(y)$.
+Then, the area under the curve between the two points $\exp(a)$ and $\exp(b)$ is:
+
+$$\int_{\exp(a)}^{\exp(b)} q(y) \, \mathrm{d}y.$$
+
+This integral should be equal to the one above, because the probability of drawing from $[a, b]$ in the original distribution should be the same as the probability of drawing from $[\exp(a), \exp(b)]$ in the transformed distribution.
+The question we have to solve here is: how do we find a function $q(y)$ such that this equality holds?
+
+We can approach this by making the substitution $y = \exp(x)$ in the first integral (see [Wikipedia](https://en.wikipedia.org/wiki/Integration_by_substitution) for a refresher on substitutions in integrals, if needed).
+We have that:
+
+$$\frac{\mathrm{d}y}{\mathrm{d}x} = \exp(x) = y \implies \mathrm{d}x = \frac{1}{y}\,\mathrm{d}y$$
+
+and so
+
+$$\int_{x=a}^{x=b} p(x) \, \mathrm{d}x
+  = \int_{y=\exp(a)}^{y=\exp(b)} p(\log(y)) \frac{1}{y} \,\mathrm{d}y
+  = \int_{\exp(a)}^{\exp(b)} q(y) \, \mathrm{d}y,
+$$
+
+from which we can read off $q(y) = p(\log(y)) / y$.
+
+In contrast, when we implemented `naive_logpdf`
+
+```{julia}
+naive_logpdf(d::MyLogNormal, y) = logpdf(Normal(d.μ, d.σ), log(y))
+```
+
+that was the equivalent of saying that $q(y) = p(\log(y))$.
+We left out a factor of $1/y$!
+
+Indeed, now we can define the correct `logpdf` function.
+Since everything is a logarithm here, instead of multiplying by $1/y$ we subtract $\log(y)$:
+
+```{julia}
+Distributions.logpdf(d::MyLogNormal, y) = logpdf(Normal(d.μ, d.σ), log(y)) - log(y)
+```
+
+and check that it works:
+
+```{julia}
+println("Sample   : $(samples_lognormal[1])")
+println("Expected : $(logpdf(LogNormal(), samples_lognormal[1]))")
+println("Actual   : $(logpdf(MyLogNormal(), samples_lognormal[1]))")
+```
+
+The same process can be applied to _any_ kind of transformation.
```

Comment:
Would you be open to restricting this a little? i.e. this doesn't work for non-invertible transformations.